__tests__/app.test.js

```diff
@@ -48,4 +48,43 @@ describe("GET /api/topic ", () => {
         expect(body.msg).toBe("bad request sorry");
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("status:200, responds with a single matching article", () => {
+    const ARTICLE_ID = 1;
+    return request(app)
+      .get(`/api/articles/${ARTICLE_ID}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: ARTICLE_ID,
```

Comment:
Well done for expecting that specific article ID to come back instead of expecting any number 👍